java-strings-2/README.MD

```diff
@@ -1,7 +1,7 @@
-## Relevant Articles
-
-- [Java Localization – Formatting Messages](https://www.baeldung.com/java-localization-messages-formatting)
-- [Check If a String Contains a Substring](https://www.baeldung.com/java-string-contains-substring)
-- [Removing Stopwords from a String in Java](https://www.baeldung.com/java-string-remove-stopwords)
-- [Blank and Empty Strings in Java](https://www.baeldung.com/java-blank-empty-strings)
-- [String Initialization in Java](https://www.baeldung.com/java-string-initialization)
+## Relevant Articles

```

Comment:
Did anything actually change in this file? If it's only whitespace or line break changes, then we need to avoid those.

Reply:
I think that Eclipse formatted this file. It looks like only whitespace changed (it wasn't my intention to change that file at all, but it got added into the commit on accident).